### ISSUE TITLE
Hello I am trying to use the start method but I get an error that is not defined when I reload the page using nextjs

### ISSUE THREAD

**Issue by willydavid1** [userflow/userflow.js#2]
Hello I am trying to use the start method but I get an error that is not defined when I reload the page using nextJS

I only want to execute the start method if a condition is met.

```
 import userflow from 'userflow.js'

 userflow.init(TOKEN)
 userflow.identify(...) //Return a Promise
 userflow.start(CODE_START) //Return a Promise
```

Error: TypeError: userflow_js__WEBPACK_IMPORTED_MODULE_1__.default.start is not a function

**Comment by sebastianseilund** [comment]
Hi @willydavid1 :)

1. Which version of userflow.js is in your node_modules?
2. Could you try to `console.log(userflow)` before the `userflow.start()` line (or run with a `debugger`) to figure out what the `userflow` object actually is. It seems strange that `init` and `identify` are there but not `start`.

**Comment by willydavid1** [comment]
1. "userflow.js": "^2.1.1" in my package.json
2. Here the object

```
 import userflow from 'userflow.js'

 userflow.init(TOKEN)
 userflow.identify(...)
 console.log('userflow', userflow)
 userflow.start(CODE_START)
```

![image](https://user-images.githubusercontent.com/55035334/118187434-86d66b00-b404-11eb-80d9-aef4c8a7a63b.png)

Am I importing userflow wrong? or do I have to wait for the identity?
something like that using async/await

```
 import userflow from 'userflow.js'

 userflow.init(TOKEN)
 (async () => {
   await userflow.identify(...)
   await userflow.start(CODE_START)
 })()
```
When I do this if it shows successfully but imagine that I have separated the `start`

I have another question I have to call init first and then start right?

**Comment by willydavid1** [comment]
As you can see on my previus response the start function appears but when I tried using debbuger as you suggest, the start funtion did not appear.

```
 import userflow from 'userflow.js'

 userflow.init(TOKEN)
 userflow.identify(...)
 console.log('userflow', userflow)
 debugger
 userflow.start(CODE_START)
```

![image](https://user-images.githubusercontent.com/55035334/118191186-d23f4800-b409-11eb-86ee-01e6939c28f7.png)

**Comment by willydavid1** [comment]
I found the error, the start method does not appear until the identify method is called and its execution ends with the await

**Comment by sebastianseilund** [comment]
Ah, my memory is slow today. This issue was fixed in v2.1.2. Please upgrade to that version :)

**Comment by sebastianseilund** [comment]
...and you don’t have to await anything before you call `userflow.start()`
